src/components/Upload.react.js

```diff
@@ -252,4 +253,4 @@ Upload.defaultProps = {
         borderColor: '#c66',
         backgroundColor: '#eee'
     }
-};
+};
```

Comment:
Also, my editor will automatically add new lines to the ends of files whereas it looks like yours removes them. Could we add these newlines back?

Reply:
Otherwise the next time I make a PR, it'll have these superfluous diffs

Reply:
Mind if I add a .editorconfig with a newline at the end of all files then?

Reply:
I think this was the codemod that made all the style changes at the end of files.